src/services/deleteTokenLogoutService.js

```diff
@@ -6,7 +6,8 @@ angular.module('openshiftCommonServices')
 
   this.$get = function($q, $injector, Logger) {
     var authLogger = Logger.get("auth");
-
+    // Using APIService creates a circular dependency
```

Comment:
Does this work?

```js
var APIService = $injector.get("APIService");
```

Reply:
Yeah good call, like `DataService` below, updating